src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/AuthorizationRule/RemoveAzureServiceBusAuthorizationRule.cs

```diff
@@ -0,0 +1,123 @@
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.ServiceBus.Commands
+{
+    /// <summary>
+    /// 'Remove-AzureRmServiceBusAuthorizationRule' Cmdlet removes/deletes AuthorizationRule
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, ServiceBusAuthorizationRuleVerb, DefaultParameterSetName = NamespaceAuthoRuleParameterSet, SupportsShouldProcess = true), OutputType(typeof(bool))]
+    public class RemoveAzureServiceBusAuthorizationRule : AzureServiceBusCmdletBase
+    {
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 0,
+            HelpMessage = "Resource Group Name.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 1, ParameterSetName = NamespaceAuthoRuleParameterSet,
+            HelpMessage = "Namespace Name.")]
+        [Parameter(ParameterSetName = QueueAuthoRuleParameterSet)]
+        [Parameter(ParameterSetName = TopicAuthoRuleParameterSet)]
+        [ValidateNotNullOrEmpty]
+        [Alias(AliasNamespaceName)]
+        public string Namespace { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 2, ParameterSetName = QueueAuthoRuleParameterSet,
+            HelpMessage = "Queue Name.")]
+        [ValidateNotNullOrEmpty]
+        [Alias(AliasQueueName)]
+        public string Queue { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 2, ParameterSetName = TopicAuthoRuleParameterSet,
+            HelpMessage = "Topic Name.")]
+        [ValidateNotNullOrEmpty]
+        [Alias(AliasTopicName)]
+        public string Topic { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 3,
+            HelpMessage = "AuthorizationRule Name.")]
+        [ValidateNotNullOrEmpty]
+        [Alias(AliasAuthorizationRuleName)]
+        public string Name { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Do not ask for confirmation.")]
+        public SwitchParameter Force { get; set; }
+
+        [Parameter(Mandatory = false)]
+        public SwitchParameter PassThru { get; set; }
+
+        public override void ExecuteCmdlet()
```

Comment:
@v-Ajnava let's consider adding a parameter for `InputObject`, so users can pass the result of the other cmdlets to this one by value.